lib/shopify_api/resources/vault_session.rb

```diff
@@ -0,0 +1,28 @@
+module ShopifyAPI
+  module VaultSessionFormat
```

Comment:
How is this different from the norman JSON format? And this should live in its own file outside of resources.